### PR TITLE
Add support for RTP Play

### PR DESCRIPTION
**Pull request by duramato**

### CODE REVIEW

**Comment by beardypig** [comment]
Looks good to me :-)

**Comment by gravyboat** [comment]
This looks good to me as well, do you want to add tests (example here: https://github.com/streamlink/streamlink/blob/master/tests/test_plugin_eurocom.py) and update the plugin matrix (example here: https://github.com/streamlink/streamlink/pull/993/files) if not no worries.

**Comment by duramato** [comment]
For sure, I'll add the tests and update the plugin matrix.

**Comment by gravyboat** [comment]
Awesome thanks @duramato!

File: tests/test_plugin_rtpplay.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import unittest
+
+from streamlink.plugins.eurocom import Eurocom
+
+
+class TestPluginEurocom(unittest.TestCase):
+    def test_can_handle_url(self):
+        # should match
+        self.assertTrue(Eurocom.can_handle_url("http://www.rtp.pt/play/"))
```
Comment (gravyboat) [inline tests/test_plugin_rtpplay.py]: Looks like this test is failing according to Travis.

Comment (duramato) [inline tests/test_plugin_rtpplay.py]: @gravyboat i'm not understanding why it's returning false

Comment (bastimeyer) [inline tests/test_plugin_rtpplay.py]: You are importing and using a different plugin (eurocom)...
Btw, the "RTPPLay" class has an unintentional capitalized L, which I think you've missed...

Comment (duramato) [inline tests/test_plugin_rtpplay.py]: Thank you.. Hindsight is 20/20, I limited my self to copy pasta it without paying much attention.

File: tests/test_plugin_rtpplay.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import unittest
+
+from streamlink.plugins.rtpplay import RTPPlay
+
+
+class TestPluginEurocom(unittest.TestCase):
```
Comment (bastimeyer) [inline tests/test_plugin_rtpplay.py]: You missed this line 😃

**Comment by gravyboat** [comment]
Looks good now, thanks @duramato!

**Comment by jalba69** [comment]
Hello, 
Not sure if something changed of it if was even supported at start but the "live" from rtp doesn't seems work , link for the direct live : 
www.rtp.pt/play/direto/rtp1
```
[Streamlink for Windows v0.8.1 - Git d7dd998]
[cli][info] Found matching plugin rtpplay for URL https://www.rtp.pt/play/direto/rtp1
[plugin.rtpplay][error] Failed to extract streams: Unable to open URL: https://streaming-live.rtp.pt/liverepeater/smil:rtp1.smil/playlist.m3u8 (403 Client Error: Forbidden for url: https://streaming-live.rtp.pt/liverepeater/smil:rtp1.smil/playlist.m3u8)
error: No playable streams found on this URL: https://www.rtp.pt/play/direto/rtp1
[End of Streamlink for Windows with ExitCode 1]
```
Thanks in advance guys.

**Pull request merged**